remote_attestation/rust/src/attestation/attested_session.rs

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::{crypto::AeadEncryptor, proto::EncryptedData};
+use anyhow::Context;
+
+pub struct AttestedSession {
+    /// Utility object for decrypting and encrypting messages using a Diffie-Hellman session key.
+    encryptor: AeadEncryptor,
+}
```

Comment:
Not sure I understand the point of this wrapper vs just using AeadEncryptor, could you add a comment? Is it just for error handling?